src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -366,12 +381,16 @@ private static State.Unit createStateUnit() {
 	 *
 	 * @param channels array of channel numbers
 	 * @param channelWidths array of channel widths (MHz)
+	 * @param txPowers array of tx powers (dBm)
 	 * @param bssids array of BSSIDs
 	 * @return the state of an AP with radios described by the given parameters
 	 */
-	public static State createState(int[] channels, int[] channelWidths, String[] bssids) {
+	public static State createState(
+		int[] channels, int[] channelWidths, int[] txPowers, String[] bssids
```

Comment:
just FYI: I will add clientRssis to the arg list in PR #51

Reply:
No problem, you can change what you like when/after you rebase :)